sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -2855,6 +2855,37 @@ def visit_vimeo(self, node):
     def visit_youtube(self, node):
         self._visit_video(node, 'https://www.youtube.com/watch?v=' + node['id'])
 
+    # ---------------------------------------------------
+    # sphinx -- extension (third party) -- sphinx-video
+    # ---------------------------------------------------
+
+    def visit_video_node(self, node):
+        autoplay = node.get("autoplay")
+        height = node.get("height")
+        width = node.get("width")
+        source_path, _ = node.get("primary_src") or (None, None)
+
+        img_key, _, _ = self.assets.add(source_path, self.docname)
+
+        if not img_key:
+            self.warn(f"Couldn't find video name {source_path}")
+            raise nodes.SkipNode
+
+        ri_filename = self._start_ri_attachment(node, img_key) + self._end_ri_attachment(node)
+
+        self.body.append(self._start_tag(node, 'div'))
+        self.body.append(self._start_ac_macro(node, 'multimedia'))
+        self.body.append(self._build_ac_param(node, 'name', ri_filename))
+        if width:
+            self.body.append(self._build_ac_param(node, "width", str(width)))
```

Comment:
(optional; tangent) I wonder if we should adjust the internals of `_build_ac_param` to always `str` cast the value -- to help simplify having to use `str` in multiple locations in the translator.